data/tekton-pipelines/windows-bios-installer-pipeline.yaml

```diff
@@ -55,6 +71,11 @@ spec:
             apiVersion: cdi.kubevirt.io/v1beta1
             kind: DataVolume
             metadata:
+              labels:
```

Comment:
What do you think about conditionally adding the kind, only if the param was set? The default will always be clusterwide.